docs/solutions/reference-designs/ad-apard32690-sl/ad-apardpfwd-sl/index.rst

```diff
@@ -0,0 +1,323 @@
+
+   SPoE PD Power Class Jumpers
+
+.. csv-table::
+   :file: SPoE_PD_Power_Class_Selection.csv
+
+.. warning::
+
+   Do not use PD Classes 15!
+
+   The :adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` evaluation board is not
+   designed to handle the class 15's power specifications.
+
+System Setup
+------------
+
+**Required Equipment**
+
+**Hardware**
+
+- :adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` Circuit Evaluation Board
+- :adi:`AD-APARD32690-SL <AD-APARD32690-SL>`
+- 10BASE-T1L media converter, either:
+
+  - :adi:`EVAL-ADIN1100EBZ <eval-adin1100>` Product Evaluation Board
+  - :adi:`AD-T1LUSB2.0-EBZ <ad-t1lusb20-ebz>` USB2.0 to 10BASE-T1L
+    Interface Board
+  - Other 10BASE-T to 10BASE-T1L media converter
+
+- Power Source, either:
+
+  - :adi:`AD-RPI-T1LPSE-SL <AD-RPI-T1LPSE-SL>` 2-port 10BASE-T1L w/SPoE
+    PSE Development Platform
+
+    - Raspberry Pi Model 3B (or higher)
+    - Micro-SD Card for Raspberry Pi
+
+  - Other 10BASE-T1L Power Coupling Network Board w/ SPoE PSE
+
+- :adi:`MAX32625PICO <MAX32625PICO>` or any other similar programmer
+  supporting the SWD interface
+
+Block Diagram
+~~~~~~~~~~~~~
+
+Setup with SPoE via PSE
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The :adi:`AD-RPI-T1LPSE-SL <AD-RPI-T1LPSE-SL>` 2-port 10BASE-T1L w/SPoE PSE
+Development
+Platform provides a complete solution for powering the
+:adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` evaluation board and the
+:adi:`AD-APARD32690-SL <AD-APARD32690-SL>` platform board via SPoE.
+
+.. figure:: apard-pfwd-block-diagram.png
+
+   Test Setup with SPoE via PSE
+
+Basic Operation
+~~~~~~~~~~~~~~~
+
+.. figure:: apard-pfwd-setup.png
+
+   Complete Evaluation Setup
+
+To establish a 10BASE-T1L connection to an
+:adi:`AD-APARD32690-SL <AD-APARD32690-SL>` using the
+:adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` evaluation board
+and ping the :adi:`AD-APARD32690-SL <AD-APARD32690-SL>`:
+
+#. Ensure that the jumpers and switches of the
+   :adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` are configured to the default settings.
+
+#. Connect the :adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` circuit evaluation
+   board to the :adi:`AD-APARD32690-SL <AD-APARD32690-SL>` Arduino headers.
+
+#. Using a USB-C cable, connect **P1** on the
+   :adi:`AD-T1LUSB2.0-EBZ <ad-t1lusb20-ebz>` evaluation board to a USB port on
+   the computer.
+
+#. Operation with SPoE PSE:
+
+   * Set the output of the PSE or DC power supply to either 24V (Class 12) or
+     55V (Class 14), depending on the settings of **JP1** and **JP2** on the
+     AD-APARDPFWD-SL board.
+   * Using a PROFIBUS cable, connect **P1** on the AD-RPI-T1LPSE-SL
+     evaluation board to **P7** on the
+     :adi:`AD-APARDPFWD-SL <AD-APARDPFWD-SL>` evaluation board.
+   * Using a PROFIBUS cable, connect **P2** on the AD-RPI-T1LPSE-SL
+     evaluation board to **P2** on the
+     :adi:`AD-T1LUSB2.0-EBZ <ad-t1lusb20-ebz>` evaluation board.
+
+#. Upload the :git-no-OS:`AD-APARD32690-SL ADIN2111 Example <projects/ASWPF/src/examples/adin2111_example/>`
```

Comment:
So, this does not exist in the no-OS main branch, cannot find a branch that relates to this and the upper case path is very suspicious, am I missing something?

Reply:
This one is missing, I'm currently working on a PR to add this project to NO-OS. You can see it [here](https://github.com/analogdevicesinc/no-OS/pull/2832). I will have to change the path today, so it will be <projects/apardpfwd>. This link was meant only as a placeholder.